components/textfield/index.css

```diff
@@ -10,116 +10,443 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    /* focus indicator is focused state */
+    &::after {
+      border-color: var(--highcontrast-textfield-focus-indicator-color, var(--mod-textfield-focus-indicator-color, var(--spectrum-textfield-focus-indicator-color)));
+      border-width: var(--mod-textfield-focus-indicator-width, var(--spectrum-textfield-focus-indicator-width));
+    }
+  }
+
+  .is-readOnly & {
+    &::after {
+      border: 0;
+    }
+  }
+
+  .spectrum-Textfield--quiet & {
+    &::after {
+      inline-size: 100%;
+      block-size: calc(100% + var(--mod-textfield-focus-indicator-gap, var(--spectrum-textfield-focus-indicator-gap)));
+      margin-inline-start: 0;
+      margin-block-start: 0;
       border-radius: 0;
+      border-color: var(--highcontrast-textfield-focus-indicator-color, var(--mod-textfield-focus-indicator-color, var(--spectrum-textfield-focus-indicator-color)));
+    }
+  }
+
+  .spectrum-Textfield--quiet.is-keyboardFocused & {
+    &::after {
+      border-width: 0 0 var(--mod-textfield-focus-indicator-width, var(--spectrum-textfield-focus-indicator-width)) 0;
     }
   }
 }
 
+/********* Child Element - Input *********/
 .spectrum-Textfield-input {
-  /* box */
   box-sizing: border-box;
-  border: var(--spectrum-textfield-texticon-border-size) solid;
-  border-radius: var(--spectrum-textfield-texticon-border-radius);
-
-  /* Apply padding by default to center text, giving consistency between input and textfield */
-  padding: var(--spectrum-textfield-texticon-padding-top)
-    var(--spectrum-textfield-texticon-padding-right) var(--spectrum-textfield-texticon-padding-bottom)
-    calc(var(--spectrum-textfield-texticon-padding-left) + 1px);
+  inline-size: 100%;
+  min-inline-size: var(--mod-textfield-min-width, var(--spectrum-textfield-min-width));
+  block-size: var(--mod-textfield-height, var(--spectrum-textfield-height));
+  padding-block-start: var(--mod-textfield-spacing-block-start, var(--spectrum-textfield-spacing-block-start));
+  padding-block-end: var(--mod-textfield-spacing-block-end, var(--spectrum-textfield-spacing-block-end));
+  padding-inline: var(--mod-textfield-spacing-inline, var(--spectrum-textfield-spacing-inline));
   /* Use padding instead of text-indent because text-indent does not left align the text in Edge browser  */
   text-indent: 0;
-
-  inline-size: 100%;
-  block-size: var(--spectrum-textfield-texticon-height);
-
   vertical-align: top; /* used to align them correctly in forms. */
+  outline: none;
+  background-color: var(--mod-textfield-background-color, var(--spectrum-textfield-background-color));
+  border: var(--mod-textfield-border-width, var(--spectrum-textfield-border-width)) solid var(--highcontrast-textfield-border-color, var(--mod-textfield-border-color, var(--spectrum-textfield-border-color)));
+  border-radius: var(--mod-textfield-corner-radius, var(--spectrum-textfield-corner-radius));
+  transition: border-color var(--mod-animation-duration-100, var(--spectrum-animation-duration-100)) ease-in-out;
```

Comment:
Should component aliases be defined for global tokens like `spectrum-animation-duration`?

Reply:
Yes, I agree with this. I added it in a new commit.